pkg/ee/cluster-backup/master/sync-controller/controller.go

```diff
@@ -167,3 +176,57 @@ func (r *reconciler) handleDeletion(ctx context.Context, log *zap.SugaredLogger,
 
 	return kuberneteshelper.TryRemoveFinalizer(ctx, r.masterClient, cbsl, cleanupFinalizer)
 }
+
+func syncCBSLCredentialSecret(
```

Comment:
Why is this not simply using the SecretReconciler from the reconciling framework?